flutter_modular/lib/src/presenters/widgets/widget_module.dart

```diff
@@ -107,8 +109,7 @@ class _ModularProviderState extends State<ModularProvider> {
   @override
   void initState() {
     super.initState();
-    // Modular.addCoreInit(widget.module);
-    _debugPrintModular("-- ${widget.module.runtimeType} INITIALIZED");
```

Comment:
Why removed from the core the debugPrint and the commented part?

Reply:
I removed the commented part as I tought they were left over from Modular 2.0 and were no longer relevant, but I could add they again.

The debugPrint was removed because the bindModule method at ModularImpl already has the "MODULE INITIALIZED" message on it. Leaving it the debugPrint here on WidgetModule would print the same message twice.